SDK_versioned_docs/version-3.0.0/widgets/swap-widget.mdx

```diff
@@ -54,16 +56,18 @@ After the user connects a wallet, the widget will use the JSON-RPC provided by t
 
 ## Installing the Widgets Library {#installing-library}
 
-To get started, install the widgets library using npm or yarn:
+To get started, install the widgets library using npm or yarn. If you don’t already use it, you’ll need to install `react-redux` as well.
 
 ```bash
 # With npm
 npm install --save @uniswap/widgets
+npm install --save react-redux
```

Comment:
nit - it'll be easier to copy/paste if you do `npm install --save @uniswap/widgets react-redux`

Reply:
Yes! I was wondering what would be easier to read, but let's optimize for developer experience and speed. I'll update, thanks!